app/services/idv/vendor_validator.rb

```diff
@@ -25,5 +27,27 @@ def idv_agent
         vendor: (idv_session.vendor || idv_vendor.pick)
       )
     end
+
+    def result
+      @_result ||= try_submit
+    end
+
+    def try_agent_action
+      yield
+    rescue => err
+      err_msg = err.to_s
+      NewRelic::Agent.notice_error(VendorValidatorError.new(err_msg))
```

Comment:
good point. I had tried just passing `err` to `notice_error` but rspec kept complaining like this:
```
  1) Verify::SessionsController user has created account #create vendor agent throws exception logs failure and re-renders form
     Failure/Error: NewRelic::Agent.notice_error(err) #VendorValidatorError.new(err_msg))
     
       NewRelic::Agent received :notice_error with unexpected arguments
         expected: (#<RuntimeError: Failed to contact proofing vendor>)
              got: (#<RuntimeError: Failed to contact proofing vendor>)
       Diff:
```